x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -55,8 +92,15 @@ export interface CaseViewTabsProps {
 
 export const CaseViewTabs = React.memo<CaseViewTabsProps>(({ caseData, activeTab }) => {
   const { features } = useCasesContext();
+  const { metricsFeatures } = useCasesFeatures();
   const { navigateToCaseView } = useCaseViewNavigation();
-  const { data: fileStatsData, isLoading } = useGetCaseFileStats({ caseId: caseData.id });
+  const { data: fileStatsData, isLoading: isFileLoading } = useGetCaseFileStats({
+    caseId: caseData.id,
+  });
+  const { data: alertStatsData, isLoading: isAlertLoading } = useGetCaseMetrics(
```

Comment:
It seems `caseData` already has `totalAlerts`. See `CaseRt` in `x-pack/plugins/cases/common/api/cases/case.ts`.

Maybe these hooks are not needed.